app/javascript/components/editor/TestsPanel.tsx

```diff
@@ -37,6 +57,7 @@ export const TestsPanel = ({
             {testTab.content}
           </code>
         </pre>
+        <div className={!reusing ? 'hidden' : ''} ref={memoTestRef}></div>
```

Comment:
In general, using `!` is slightly less readable than its logical inverse:

```suggestion
        <div className={reusing ? '' : 'hidden'} ref={memoTestRef}></div>
```